app/models/person.rb

```diff
@@ -288,6 +288,10 @@ def first_name_with_initial
   # Deprecated: This is view logic (how to display name) and thus should not be in model layer
   # Consider using PersonViewUtils
   def name(community_or_display_type=nil)
+    # TODO Make `community_or_display_type` param mandatory.
+    # The custom error is here in order to get better error message while, but it can be removed in the future.
+    raise ArgumentError.new("Missing community or display type. Can not know how to format the name") if community_or_display_type.nil?
+
```

Comment:
Just to get better error messages. I have a hunch that when we deploy this to production, we will get a couple of those errors and that's why it's nicer to have better and more explicit error message than "Wrong number of arguments 0 for 1)